src/EmployerAan/SFA.DAS.EmployerAan.Api/Controllers/MemberProfilesController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerAan.Application.Employer.Queries.GetEmployerMemberSummary;
+using SFA.DAS.EmployerAan.Application.MemberProfiles.Queries.GetMemberProfileWithPreferences;
+using SFA.DAS.EmployerAan.Infrastructure;
+using SFA.DAS.EmployerAan.Models;
+
+namespace SFA.DAS.EmployerAan.Api.Controllers;
+
+[Route("Members")]
```

Comment:
All the routes should be in lower case

Reply:
This is missing ApiController attribute

Reply:
This is fixed in codereview ticket, but will fix it here as well.